### PR TITLE
Migrating to SimpleLogin

### PR DESCRIPTION
**Pull request by MysticalPvE**
Changed MX records and TXT records to migrate from improvmx to simplelogin.
Added verification TXT record and SPF. Added MX records of SimpleLogin for forwarding.

### CODE REVIEW

File: domains/dhilipan.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   },
   "record": {
     "A": ["75.2.60.5"],
-    "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
-    "TXT": "v=spf1 include:spf.improvmx.com ~all"
+    "MX": ["mx1.simplelogin.co.", "mx2.simplelogin.co."],
```
Comment (CuteDog5695) [inline domains/dhilipan.json:9]: Dots aren't necessary for MX records.
```suggestion
    "MX": ["mx1.simplelogin.co", "mx2.simplelogin.co"],
```

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/dhilipan.json (hunk 1 above)
Comment (MysticalPvE) [reply domains/dhilipan.json:9]: > Dots aren't necessary for MX records.

Simplelogin docs insist on it.

**Comment by MysticalPvE** [comment]
Conversations resolved and 1 commit updated. Can I get this merged?

**Comment by CuteDog5695** [comment]
> Conversations resolved and 1 commit updated. Can I get this merged?

Checks failed for the testing part, commited that thing for ya now

**Review by iostpa — APPROVED** [review]
This was the fix btw. Thats why tests failed.

**Pull request merged**